src/stream_compression.jl

```diff
@@ -32,7 +32,7 @@ Creates an LZ4 compression codec.
     32-bits checksum of decompressed data
 - `frametype::FrameType=normal_frame)`:  `normal_frame` or `skippable_frame`
 - `contentsize::Integer=0`: Size of uncompressed content (0 for unknown)
-- `blockchecksumflag::Bool=false`: if `true`, each block is followed by a
+- `blockchecksum::Bool=false`: if `true`, each block is followed by a
```

Comment:
You changed the flag here, but below at line 56 it still references `blockchecksumflag`

Reply:
This has been fixed and tests have been added for it